engine/classes/Elgg/UserCapabilities.php

```diff
@@ -244,13 +244,13 @@ public function canEditAnnotation(ElggEntity $entity, $user_guid = 0, ElggAnnota
 	 * Can a user add an entity to this container
 	 *
 	 * @param ElggEntity $entity    Container entity
-	 * @param int        $user_guid The GUID of the user creating the entity (0 for logged in user).
 	 * @param string     $type      The type of entity we're looking to write
 	 * @param string     $subtype   The subtype of the entity we're looking to write
+	 * @param int        $user_guid The GUID of the user creating the entity (0 for logged in user).
 	 *
 	 * @return bool
 	 */
-	public function canWriteToContainer(ElggEntity $entity, $user_guid = 0, $type = 'all', $subtype = 'all') {
+	public function canWriteToContainer(ElggEntity $entity, string $type, string $subtype, int $user_guid = 0) {
```

Comment:
do we need to document the order change? we documented other changes from this class in the upgrade notes

Reply:
removed other docs